### PR TITLE
Add werror/rerror to block drive

### PR DESCRIPTION
**Pull request by pezhang**
1. Add werror/rerror to readonly data disk

ID:1321735

Signed-off-by: Pei Zhang <pezhang@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/readonly_disk.cfg [hunk 1]
```diff
@@ -7,7 +7,14 @@
     image_name_data = "images/data_disk"
     image_size_data = 1G
     force_create_image_data = yes
-    copy_cmd = copy /Y %s %s
-    src_file = WIN_UTIL:\README
-    # Please make the disk_letter the same in create_partition_cmd and format_cmd
+    copy_cmd = copy / Y % s % s
```
Comment (xutian) [inline qemu/tests/cfg/readonly_disk.cfg]: wrong command

File: qemu/tests/cfg/readonly_disk.cfg [hunk 2]
```diff
@@ -7,7 +7,14 @@
...
+    src_file = WIN_UTIL:
```
Comment (xutian) [inline qemu/tests/cfg/readonly_disk.cfg]: invalid syntax

File: qemu/tests/cfg/readonly_disk.cfg [hunk 3]
```diff
@@ -7,7 +7,14 @@
...
+        README
+    # Please make the disk_letter the same in create_partition_cmd and
+    # format_cmd
     disk_letter = I
+    variants:
+        - @default:
+        - with_wrerror:
+            drive_werror = stop
```
Comment (xutian) [inline qemu/tests/cfg/readonly_disk.cfg]: does IDE interface support werror action?

Comment (pezhang) [inline qemu/tests/cfg/readonly_disk.cfg]: IDE does not support. 'no ide, ahci' has been set in the public params.

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by CongSmile** [comment]
-drive id=drive_cd1,if=none,rerror=stop,werror=stop,snapshot=off,aio=native,media=cdrom,file=/usr/share/avocado/data/avocado-vt/isos/windows/winutils.iso \
    -device ide-cd,id=cd1,drive=drive_cd1,bootindex=2,bus=ide.0,unit=0 \

**Comment by pezhang** [comment]
Code has been updated. @CongSmile @xutian . 
Could you help review it again? Thanks.

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
looks good to me.

**Pull request merged**